src/shared/ui/SearchForm/SearchForm.css

```
@@ -0,0 +1,34 @@
+.searcher {
+    justify-content: center;
+    display: flex;
+    gap: 6px;
+}
+
+.searcher-input {
+    width: 713px;
+    Height: 57px;
+    border-radius: 4px;
+    background-color:#424242;
```

```suggestion
    background-color: #424242;
```
By the way, all this refactoring is usually fixed by `eslinter` or `prettier`

---

I have fixed with prettier